lib/pages/chat/chat_event_list.dart

```diff
@@ -150,7 +149,7 @@ class ChatEventList extends StatelessWidget {
                     key: ValueKey(event.eventId),
                     index: index,
                     controller: controller.scrollController,
-                    highlightColor: LinagoraRefColors.material().primary[99],
+                    highlightColor: Theme.of(context).highlightColor,
```

Comment:
IMO: - This is a bad idea.

- Why do you use this color?
- Need to confirm with designer, team?
- Compared to the old color, I find it even worse.

Reply:
`Theme.of(context).highlightColor` is based on default color of material design 3, there is no design for this from beginning. And change to global color, later we only need to change in one place (ThemeData file).

Reply:
Tested on android and web LGTM.
Also discussed with Bob, looks good for now as this screen will evolve in near future